app/graphql/mutations/create_model.rb

```diff
@@ -0,0 +1,42 @@
+module Mutations
+  class CreateModel < BaseMutation
```

Comment:
I suggest to put all mutations related with single resource in separate namespace(`models`). It's more readable IMO.